lib/ruby_speech/grxml/grammar.rb

```diff
@@ -116,9 +116,18 @@ def inline
       # @return self
       #
       def inline!
-        xpath("//ns:ruleref", :ns => GRXML_NAMESPACE).each do |ref|
-          rule = rule_with_id ref[:uri].sub(/^#/, '')
-          ref.swap rule.dup.children
+        loop do
+          rule = nil
+          xpath("//ns:ruleref", ns: GRXML_NAMESPACE).each do |ref|
+            rule = rule_with_id ref[:uri].sub(/^#/, "")
+            unless rule
+              raise
+                ArgumentError,
```

Comment:
> please enlighten me as to why that is

Heh, I'm really not sure why this triggers `dynamic constant assignment`.  Unfortunately, the following proper argument alignment will not suffice because it causes the line with the 2nd argument to exceed 80 characters:

```ruby
          unless rule
            raise ArgumentError,
                  "The Ruleref \"#{ref[:uri]}\" is referenced but not defined"
          end
```

I was then optimistic that hound-ci would accept non-aligned arguments based on the following rule exception in ruby style guide that it is based off of:

> Align the parameters of a method call if they span more than one line. **When aligning parameters is not appropriate due to line-length constraints, single indent for the lines after the first is also acceptable.**
https://github.com/bbatsov/ruby-style-guide#no-double-indent

But alas, hound-ci does not accept this either:

```ruby
          unless rule
            raise ArgumentError,
              "The Ruleref \"#{ref[:uri]}\" is referenced but not defined"
          end
```

In lieu of hound-ci honoring that exception to the rule, I believe I'll need to assign `"The Ruleref \"#{ref[:uri]}\" is referenced but not defined"` to a separate variable in order to appease.